lib/publiccloud/instance.pm

```diff
@@ -148,7 +148,11 @@ sub _prepare_ssh_cmd {
         $cmd = "\$'$cmd'";
     }
 
-    my $ssh_cmd = sprintf('ssh -E /var/tmp/ssh_sut.log %s "%s@%s" -- %s', $args{ssh_opts}, $args{username}, $self->public_ip, $cmd);
+    my $ssh_cmd;
+    my $ssh_log_file;
+    $ssh_log_file = "-E /var/tmp/ssh_sut.log" if ($args{ssh_opts} !~ /-E \/var\/tmp\/ssh_sut\.log/);
```

Comment:
Kind suggetion:
```suggestion
    my $ssh_cmd;
    my $log = '/var/tmp/ssh_sut.log';
    my $ssh_log_file;
    $ssh_log_file = "-E $log" if ($args{ssh_opts} !~ m{-E\s+$log});
```

Reply:
in perl regexpr we can use different separator than '/', to avoid escaping it.

Reply:
Also, suggested a more VR as negative-test, pre-adding the log string, to see that it still works.

Reply:
If the log name is in `$log`, I'd say it's better for readability to do the regexp as:
```
    $ssh_log_file = "-E $log" if ($args{ssh_opts} !~ m/-E\s+$log/);
```
It's more common.

OTOH, if the log name is not in a variable as in the original code, then better to use a different separator for the regexp to avoid escaping the `/` characters:
```
    $ssh_log_file = "-E /var/tmp/ssh_sut.log" if ($args{ssh_opts} !~ m|-E /var/tmp/ssh_sut\.log|);
```

Reply:
And wdyt about shorter form in https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/21019#discussion_r1925509155 ?

Reply:
Fewer lines are always good IMO ;)